packages/fast-components-react-base/src/stack-panel/stack-panel.props.ts

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import {
+    ManagedClasses,
+    StackPanelClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface StackPanelManagedClasses
+    extends ManagedClasses<StackPanelClassNameContract> {}
+export interface StackPanelUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+export enum stackPanelOrientation {
+    horizontal = "horizontal",
+    vertical = "vertical",
+}
+
+export interface ItemSpanOverride {
+    [key: number]: number;
+}
+
+export interface StackPanelHandledProps extends StackPanelManagedClasses {
+    /**
+     * Whether items outside the viewport should be rendered or not
+     * default is true
+     */
+    enableVirtualization?: boolean;
+
+    /**
+     *  the indexes of children that should always be rendered
```

Comment:
Can you provide a use case?

Reply:
The scenario you can see working in the stories is the case where we always render the first and last item in the list so that tabbing to the component brings one of those into focus.  The other use case is if someone wanted to implement something that resembled lazy loading where some items don't unload after they have been loaded.